### PR TITLE
Ond211 1841 handling custom platforms messages

### PR DESCRIPTION
**Pull request by gaborod16**
### New Features

 * [OND211-1841] Add custom placeholder platforms to the Message.Platform Enum
 * [OND211-1841] Add a simple CRUD to handle the custom name mapping for Placeholder platforms (per Agent)

### CODE REVIEW

File: ondewo/nlu/agent.proto [hunk 1]
```diff
@@ -945,8 +949,34 @@ message ModelStatus {
...
 
 message GetModelStatusesResponse {
-    // Required. The project that the agent to fetch is associated with.
-    // Format: `projects/<Project ID>`.
     repeated ModelStatus model_statuses = 1;
 }
 
+message CustomPlatformInfo {
+    // Required. ID of the Platform.
+    // Custom Intent.Message.Platform placeholder. Note: it will not work with non-placeholders values
```
Comment (Phips0812) [inline ondewo/nlu/agent.proto:957]: This is a good note :D

**Review by Phips0812 — APPROVED** [review]
Good Work!

**Review by teddius — APPROVED** [review]
Looks good 🙂

**Review by teddius — APPROVED** [review]
Looks good 😊

Minor style remark: in some files the indentation has been changed so it looks like that there are a lot of changes which they are not. Maybe better to change it back so it is clear on which files changes occurred in this PR.

**Pull request merged**